sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FunMatcher.kt

```diff
@@ -89,19 +94,13 @@ class FunMatcherImpl(
             checkCallParameters(functionDescriptor)
 
     private fun checkTypeOrSupertype(functionDescriptor: CallableDescriptor) =
-        qualifier.isNullOrEmpty() && definingSupertype.isNullOrEmpty() ||
```

Comment:
Also, if you changed the FunMatcher, you should add an appropriate test for it

Reply:
Fixed.